frontend/src/utils/tourService/tourUtils.ts

```diff
@@ -1,15 +1,28 @@
-const baseUrl = import.meta.env.VITE_API_PATH;
+interface TourFilters {
+  location: string;
+  accommodationType: string;
+  days: string;
+  budget: string;
+  safariType: string;
+}
+
+const baseUrl = import.meta.env.VITE_BASE_URL;
```

Comment:
the baseUrl const be as follows.
```
const baseUrl = import.meta.env.VITE_API_PATH
```
As i understood it the VITE_BASE_URL is for frontend routing while VITE_API_PATH is for api for servers